Services.Test/Simulation/JavascriptInterpreterTest.cs

```diff
@@ -41,14 +44,16 @@ public JavascriptInterpreterTest(ITestOutputHelper log)
         public void ReturnedStateIsIntact()
         {
             // Arrange
+            InternalDeviceState deviceState = new InternalDeviceState();
+
             var filename = "chiller-01-state.js";
             var context = new Dictionary<string, object>
             {
                 ["currentTime"] = DateTimeOffset.UtcNow.ToString("yyyy-MM-dd'T'HH:mm:sszzz"),
                 ["deviceId"] = "device-123",
                 ["deviceModel"] = "room"
             };
-            var state = new Dictionary<string, object>
+            var sensors = new Dictionary<string, object>
```

Comment:
please revert this, generally speaking the `state` variable contains also non-sensors data